tendrl/gluster_integration/message/callback.py

```diff
@@ -526,9 +526,43 @@ def volume_delete(self, event):
                                 brick_path,
                                 recursive=True
                             )
+                            # Delete brick dashboard from grafana
+                            job_id = monitoring_utils.update_dashboard(
+                                "%s|%s" % (event['message']['name'],
+                                           brick.key.split('/')[-1].replace("_","/")
+                                           ),
+                                RESOURCE_TYPE_BRICK,
+                                NS.tendrl_context.integration_id,
+                                "delete"
+                            )
+                            logger.log(
+                                "debug",
+                                NS.publisher_id,
+                                {
+                                    "message": "Update dashboard job %s "
```

Comment:
Add brick name, integration id  in log message ? Current log message is not useful

Reply:
i will change